tests/Client.UiTests/Runner.fs

```diff
@@ -0,0 +1,22 @@
+module Client.UiTests.Runner
+
+open canopy.classic
+open canopy.types
+open Expecto
```

Comment:
I am curious about Expecto, but honestly I think it would be confusing to have different testing frameworks in one project, so let's switch to xUnit for now :) Looks like definitely possible, e.g. [here](https://www.olivercoding.com/2018-06-29-canopy/)

Reply:
this link is quite old, I've tried to match it with current api, but it seems I won't be able to run it the same way

Reply:
Sooooo... this seems to work (instead of both Runner and Tests):
```fsharp
module Client.UiTests.Tests

open canopy.classic
open canopy.types
open System
open Xunit

type Fixture() =
    do
        //start Chrome // Use this if you want to see your tests in the browser
        start ChromeHeadless
        resize (1280, 960)

let serverUrl = Environment.GetEnvironmentVariable("SERVER_URL")

let startApp() =
    url serverUrl
    waitForElement("#elmish-app")

type Tests() = 
    interface IClassFixture<Fixture>        

    [<Fact>]
    member _.``Soundcheck - server is online``() =
        startApp()
        
    [<Fact>]
    member _.``Validate word is loaded``() =
        startApp()
        
        click (first ".link-block")
        
        let taskText = (element ".task-label").GetAttribute("innerText")
        
        Assert.NotNull taskText
```

Reply:
does this `Fixture` make it run only once? I'll modify the code then or do you have a working config already?

Reply:
Fixture code runs only once prior to running all tests.